### ISSUE TITLE
Remove backup/restore notification for verified wallet users

### ISSUE THREAD

**Issue by mandar-brave** [brave/brave-browser#6259]
backup functionality is now limited, and more users keep their BAT using a Custodial wallet, so we should only show backup notification once after 30 days of usage. If user connects uphold KYC wallet we should never display it.

**Comment by NejcZdovc** [comment]
@mandar-brave why? user could still have ugp in their wallet and they would loose it. Also payment id is connected to ads earnings

**Comment by ethanbb** [comment]
For me this keeps showing up again and again even though I backed up my wallet a long time ago, i.e. copied the recovery key to a secure note in LastPass. I also have a verified wallet with Uphold. Is this a bug or what?

**Comment by mandar-brave** [comment]
@ethanbb the reason it shows it because it does that at a once a month cadence; this is just to ensure user doesn't forget.

@NejcZdovc we should have a never show again option; lets chat and design UX accordingly.

**Comment by PandaWhisperer** [comment]
Sorry for hijacking this thread, but is there a way to turn off the backup reminder (or change its frequency)? I've got mine backed up securely and the constant reminders are getting annoying.

**Comment by mandar-brave** [comment]
@NejcZdovc as backup functionality is now limited, and more users keep their BAT using a Custodial wallet, please go ahead and limit this to once to get user acquainted.

**Comment by LaurenWags** [comment]
Verified passed with

```
Brave | 1.14.72 Chromium: 85.0.4183.83 (Official Build) dev (64-bit)
-- | --
Revision | 94abc2237ae0c9a4cb5f035431c8adfb94324633-refs/branch-heads/4183@{#1658}
OS | macOS Version 10.14.6 (Build 18G3020)
```

Verified **`Case 1`** from https://github.com/brave/brave-core/pull/6344 
 - Confirmed after updating `Preferences` file as directed and relaunching, the backup notification was shown after 5s. 
 - Confirmed after clicking on the `x` icon, the notification did not show again after 5s.
 - Closed and relaunched. Confirmed notification again did not show after 5s.
 - Set date forward on my computer by 1 month and launched. Confirmed notification did not show after 5s.

Verified **`Case 2`** from https://github.com/brave/brave-core/pull/6344
 - Confirmed after updating `Preferences` file as directed and relaunching, the backup notification was not shown after 5s. 
 - Closed and relaunched. Confirmed notification again did not show after 5s.
 - Set date forward on my computer by 1 month and launched. Confirmed notification did not show after 5s.

for reference, this is how I updated my `Preferences` file for both cases per the PR:

<img width="334" alt="Screen Shot 2020-09-02 at 3 15 41 PM" src="https://user-images.githubusercontent.com/28145373/92027938-3a914200-ed31-11ea-9fc9-dece0dec6983.png">

----------
